### PR TITLE
Cherry-picks for 30

### PR DESCRIPTION
**Pull request by tytan652**
### Description

TLDR: Cherry-picked translation, bug fixes and merged PRs placed on the 30 milestone.

Cherry picks for 30:
- Translation update https://github.com/obsproject/obs-studio/commit/0d450a34a0564c0c61788b8b1da16934353d4b2f
- https://github.com/obsproject/obs-studio/pull/9688
- https://github.com/obsproject/obs-studio/pull/9690
- https://github.com/obsproject/obs-studio/pull/9687
- https://github.com/obsproject/obs-studio/pull/9693
- https://github.com/obsproject/obs-studio/pull/9696
- https://github.com/obsproject/obs-studio/pull/9695
- https://github.com/obsproject/obs-studio/pull/9679
- https://github.com/obsproject/obs-studio/pull/9706
- https://github.com/obsproject/obs-studio/pull/9620
- https://github.com/obsproject/obs-studio/pull/9713
- https://github.com/obsproject/obs-studio/pull/9555
- https://github.com/obsproject/obs-studio/pull/9709
- https://github.com/obsproject/obs-studio/pull/9721
- https://github.com/obsproject/obs-studio/pull/9708
- https://github.com/obsproject/obs-studio/pull/9698
- https://github.com/obsproject/obs-studio/pull/9717
- https://github.com/obsproject/obs-studio/pull/9724
- https://github.com/obsproject/obs-studio/pull/9746
- https://github.com/obsproject/obs-studio/pull/9743
- https://github.com/obsproject/obs-studio/pull/9745
- https://github.com/obsproject/obs-studio/pull/9712

### Motivation and Context

Cherry-pick commits meant for master and 30.

### How Has This Been Tested?

Did not tested personally.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

- Tweak (non-breaking change to improve existing functionality)

- Code cleanup (non-breaking change which makes code smaller or more readable)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [ ] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by kkartaltepe** [comment]
#9743 will directly lead to crashes without #9745 so we should pull both or neither.

**Comment by tytan652** [comment]
So the latter needs to be merged on master, and put in this PR.

**Comment by tytan652** [comment]
#9745 added

**Comment by RytoEX** [comment]
Please use `git cherry-pick -x` so that the commit messages retain a link to the source commits.

Please also include these in the appropriate order:
* #9687 (https://github.com/obsproject/obs-studio/commit/c96111ab042a94a25d805546e9425d148e627d1c)
* #9679 (https://github.com/obsproject/obs-studio/commit/44f3f7929678292f2680f1962ceddefa9766d590)

For the former, I prefer to keep files like services.json in sync as appropriate. For the latter, it's a minor code cleanup.

**Comment by tytan652** [comment]
Should be good now.

**Comment by RyzenDew** [comment]
can #9475 and #8832 be added as well both are tested and work without issues.

**Comment by gxalpha** [comment]
@RyzenDew No, they cannot. We're only adding merged bug fixes. The PRs above are neither merged nor bug fixes.

**Comment by tytan652** [comment]
#9712 added

**Comment by RytoEX** [comment]
Still unsure if #9620 was meant to be included. @GeorgesStavracas @kkartaltepe ?

Otherwise, looks fine.

**Comment by GeorgesStavracas** [comment]
@RytoEX fine to keep #9620 here IMO

**Comment by RytoEX** [comment]
> @RytoEX fine to keep #9620 here IMO

It's not included yet, which is what I meant to ask.

@tytan652 Can you include #9620? The PR title is misleading, as it used to be adding a toggleable option, but was changed to a single-line change that seems to be a bug-fix.

**Review by RytoEX — APPROVED** [review]
Looks good to me.
